### PR TITLE
add a default linux config everyplace there is a windows/darwin config

### PR DESCRIPTION
**Pull request by cojam-dutchie**
This came up once I started using these tasks in the deployment pipeline.

### CODE REVIEW

File: tools/yamlfmt/task_linux.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    vars:
+      TEMP_DIR:
+        sh: mktemp -d
+      # YAMLfmt uses x86_64 instead of amd64
+      FIXED_ARCH: '{{if eq ARCH "amd64"}}x86_64{{else}}{{ARCH}}{{end}}'
+    cmds:
+      - defer: rm -rf {{.TEMP_DIR}}
+      - curl -Lo {{.TEMP_DIR}}/yamlfmt.tar.gz https://github.com/google/yamlfmt/releases/download/v0.9.0/yamlfmt_0.9.0_Linux_{{.FIXED_ARCH}}.tar.gz
+      - tar -xvf {{.TEMP_DIR}}/yamlfmt.tar.gz -C {{.TEMP_DIR}}
+      - sudo mv {{.TEMP_DIR}}/yamlfmt /usr/local/bin/yamlfmt
```
Comment (anthonyserious) [inline tools/yamlfmt/task_linux.yaml:17]: Add a command to delete the .tar.gz file?

**Review by anthonyserious — CHANGES_REQUESTED** [review]
I really don't know if this will be fine, tbh.  But, I think it would only fail if you try to use a linux task with an empty set of tasks.

Also, you might need to change those empty task sets from `tasks:` to `tasks: {}`.

File: tools/yamlfmt/task_linux.yaml (hunk 1 above)
Comment (cojam-dutchie) [reply tools/yamlfmt/task_linux.yaml:17]: I mostly just copied this from `task_darwin.yaml` but swapped "Darwin" for "Linux". That being said, I think `defer: rm -rd {{.TEMP_DIR}}` should clean up everthing.

**Comment by cojam-dutchie** [comment]
> I really don't know if this will be fine, tbh. But, I think it would only fail if you try to use a linux task with an empty set of tasks.
> 
> Also, you might need to change those empty task sets from `tasks:` to `tasks: {}`.

These are copied from the `task_windows.yaml` and have been tested somewhat (I am running the yaml and python steps in github pipelines).

**Comment by anthonyserious** [comment]
> > I really don't know if this will be fine, tbh. But, I think it would only fail if you try to use a linux task with an empty set of tasks.
> > Also, you might need to change those empty task sets from `tasks:` to `tasks: {}`.
> 
> These are copied from the `task_windows.yaml` and have been tested somewhat (I am running the yaml and python steps in github pipelines).

Oh yeah, in that case 👍

**Review by anthonyserious — APPROVED** [review]
👍

**Pull request merged**